### PR TITLE
Knowledge: Capability to perform GDPR question answering. Signed-off-…

### PR DESCRIPTION
**Pull request by nishithsharma9**
Nishith Sharma <ns5014@nyu.edu>

**Describe the contribution to the taxonomy**
- The user will ask if the given scenario adheres to GDPR Compliance. 
- Answer that based on the GDPR compliance guidelines.
-  Provide the reasoning behind it and respond with relevant citation to answer the given question.

**Input given at the prompt**
```
Does the GDPR require controllers to obtain explicit consent from data subjects for processing sensitive data?
```

**Response from the original model**
```
No, the GDPR does not require explicit consent for processing sensitive data. Instead, it introduces higher standards for consent, including the requirement that consent must be given freely, specifically, and informed. However, for sensitive data, the GDPR requires a higher legal basis for processing, such as explicit consent or another appropriate justification.
```

**Response from the fine-tuned model**
```
Yes, the GDPR does require controllers to obtain explicit consent from data subjects for processing sensitive data. This is because sensitive data is defined as personal data which reveals racial or ethnic origin, political opinions, religious or philosophical beliefs, or trade union membership, and genetic data, biometric data for the purpose of uniquely identifying a natural person, data concerning health or data concerning a natural person's sex life or sexual orientation. Explicit consent means that the data subject has given a freely given, specific, informed, and unambiguous indication of his or her agreement to the processing of personal data relating to him or her for one or more specific purposes. The GDPR also requires controllers to be able to demonstrate that the data subject has provided valid consent, and that consent can be withdrawn at any time. The GDPR also requires controllers to provide data subjects with clear, transparent, and easily understandable information about how their personal data will be processed, and the controller's legal basis for processing that data, including any third-party disclosures or international transfers. The GDPR also requires controllers to ensure that data subjects' rights are respected, including the right to access, rectification, erasure, restriction of processing, data port
```

**Contribution checklist**

- [x] The contribution was tested with `ilab generate`
- [x] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/compliance/gdpr_complaince/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+created_by: Nishith Sharma
```
Comment (jjasghar) [inline knowledge/compliance/gdpr_complaince/qna.yaml:1]: Please put your GitHub username here.

File: knowledge/compliance/gdpr_complaince/qna.yaml
Comment (jjasghar) [inline knowledge/compliance/gdpr_complaince/qna.yaml:1]: We are missing an `attribution.txt` here also.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix these issues including the DCO.

**Comment by nishithsharma9** [comment]
Tagging here for reference: @adampingel

**Pull request closed without merging**